rotkehlchen/tests/unit/accounting/evm/test_paraswap.py

```diff
@@ -114,8 +197,11 @@ def test_paraswap_swap_with_fee(accountant: 'Accountant'):
             price=Price((FVal(swap_amount_str) + FVal(fee_amount_str)) / FVal(receive_amount_str)),
             pnl=PNL(taxable=ZERO, free=ZERO),
             cost_basis=None,
-            index=2,
             extra_data=extra_data,
         )]
     expected_processed_events[0].count_cost_basis_pnl = True  # since it's not settable at ctor
+    expected_processed_events[1].count_cost_basis_pnl = True  # since it's not settable at ctor
+    expected_processed_events[2].count_cost_basis_pnl = True  # since it's not settable at ctor
     assert pot.processed_events == expected_processed_events
+    assert pot.pnls[AccountingEventType.TRANSACTION_EVENT].taxable == events[0].balance.amount + events[1].balance.amount  # noqa: E501
```

Comment:
Why do you think this is correct? So the PnL of this entire action is amount of the first 2 receivals? What about the fee? I do not understand what this test is trying to test and as a reader it's impossible to understand what you are trying to test for